src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -128,7 +128,7 @@ await using (var context = InMemoryApplicationDbContext(contextId))
             {
                 var actual = await context
                     .ReleaseVersions
-                    .SingleAsync(rv => rv.PublicationId == publication.Id);
```

Comment:
This is undoing a change made by the Release to ReleaseVersion renaming work, probably from resolving merge conflicts.